validator-cli/src/utils/ethers.ts

```
@@ -10,67 +11,103 @@ import {
+  }
+};
+const getTransactionHandler = (chainId: number, network: Network) => {
   switch (chainId) {
     case 11155111:
-      return ArbToEthTransactionHandler;
+      switch (network) {
+        case Network.DEVNET:
+          return ArbToEthDevnetTransactionHandler;
+        case Network.TESTNET:
+          return ArbToEthTransactionHandler;
+      }
```

Biome (1.9.4)

[error] This case is falling through to the next case.

Add a `break` or `return` statement to the end of this case to prevent fallthrough.

(lint/suspicious/noFallthroughSwitchClause)